js/react/components/nav/index.jsx

```diff
@@ -47,45 +43,32 @@ function Nav(props) {
 								Community
 							</a>
 						</li>
-						{/* No user */}
-						{!user && (
+						{!user ? (
```

Comment:
Please changes this back to what it was before

Nested ternary conditions are not readable and overall a bad convention

@NMilivojevic did it the same way as you do now and I requested from her to change it as well

Reply:
Ok I will separate them, but the text display before was not what we wanted, so I changed it.